### PR TITLE
Pull request for minor changes

### PR DESCRIPTION
**Pull request by aselvan**
Hello @marcopaganini ,
First, thank you for this handy command-line program to run speed test. This pull request (if you want to accept) is to provide a progress while doing the download. Also made minor changes to defaults and flag name. The following is the change commit message.

_`"Added progress meter, changed default for re-run interval to 2 minutes. Also fixed to do nothing when run w/ in the re-run interval."`_

### CODE REVIEW

File: flags.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func (x *cmdLineOpts) parseFlags() error {
 	flag.StringVar(&x.location, "location", defaultLocation, "TestMyNet location")
 	flag.IntVar(&x.datasize, "size", defaultDataSize, "Test size in KBytes")
 	flag.BoolVar(&x.dryrun, "dry-run", false, "Dry-run mode")
-	flag.BoolVar(&x.force, "I-WANT-TO-GET-BANNED", false, "Allow program to hit testmy.net more often than it should.")
```
Comment (marcopaganini) [inline flags.go]: When I wrote this little utility, I contacted testmy.net asking if they'd be interested in listing this as a free tool for their users. They were not that interested, and one of the main reasons is that they were afraid people would unwittingly overload their servers. I promised I'd make it as hard as possible to have it happening. This weirdly named flag makes it clear to people that it should only be considered in extreme cases, whereas "--force" seems like something without much consequence.

**Review by marcopaganini — CHANGES_REQUESTED** [review]
Nice changes, thanks (with one minor nit, please see comments). Also, did you run `gofmt` in your code? It's hard to see in the github editor/viewer, but I see things that appear to have the wrong indentation.

File: flags.go (hunk 1 above)
Comment (aselvan) [reply flags.go]: Thanks for clarification. I did not realize you were working with testmy.net to list this as approved tool. And I also understand why testmy.net is reluctant. I am fine with changing it back, let me know. My only reason to change it to match it like a normal flag. If someone is going to try it often, they are going to do regardless of how it is spelled :)

**Comment by marcopaganini** [comment]
Hi Arul. Yes, I pinged them when I wrote the tool (a while back) but they
were not interested in a tool, since their primary method is using the web
browser. Since I already had the tool, I left it for others to use (I use
it). The flag in capitals and saying "I-WANT-TO-GET-BANNED" should be
enough warning for those that might think this is a casual flag. The word
"Banned" will at least bring some thought into using it :)

PS: Don't forget to run `gofmt` in your code! All the other changes are
great!

**Comment by aselvan** [comment]
> Hi Arul. Yes, I pinged them when I wrote the tool (a while back) but they were not interested in a tool, since their primary method is using the web browser. Since I already had the tool, I left it for others to use (I use it). The flag in capitals and saying "I-WANT-TO-GET-BANNED" should be enough warning for those that might think this is a casual flag. The word "Banned" will at least bring some thought into using it :) PS: Don't forget to run `gofmt` in your code! All the other changes are great!

Sounds good. I used vi to edit and I am sure it messed up the tabs. Let me run gofmt and commit the changes and will also change back to the scary flag :) BTW: This was my first attempt to do anything w/ golang and seems like its pretty neat. I am old school java/c++/asm guy from the 90's :)

**Review by marcopaganini — APPROVED** [review]

File: main.go [hunk 2]
```diff
@@ -14,21 +14,47 @@ import (
...
+	"strings"
 	"time"
 
+	"github.com/dustin/go-humanize"
 	"github.com/marcopaganini/logger"
 )
 
 const (
 	stateFile          = ".testmynet-cli.state"
-	minDurationMinutes = 15
```
Comment (marcopaganini) [inline main.go]: Oh, the 15 minutes was also a number that came out of testmy.net (IIRC), so it should stay.

**Review by marcopaganini — CHANGES_REQUESTED** [review]
Just one more comment (Just saw it now) about the 15m vs 2m

File: main.go (hunk 2 above)
Comment (aselvan) [reply main.go]: Got it reverted back to 15.

**Review by marcopaganini — APPROVED** [review]

**Comment by marcopaganini** [comment]
Thanks for the PR! Happy new year! :)

**Comment by aselvan** [comment]
You are welcome and Happy new year to you as well.

**Pull request merged**